man/corosync.conf.5

```diff
@@ -390,9 +395,15 @@ The default is 650 milliseconds.
 .TP
 token_retransmit
 This timeout specifies in milliseconds after how long before receiving a token
-the token is retransmitted.  This will be automatically calculated if token
-is modified.  It is not recommended to alter this value without guidance from
-the corosync community.
+the token is retransmitted. If not specified in config file, it will be
+automatically calculated if token or token_retransmits_before_loss_const is set (explained in
+.B token_retransmits_before_loss_const
+and
+.B
+token).
+
+The hold will be automatically calculated if token_retransmit is modified. It is not
+recommended to alter this value without guidance from the corosync community.
 
 The default is 238 milliseconds.
```

Comment:
This one needs change, because it depends on token timeout which depends on number of nodes -> for 2 nodes it is really 238, but for 3 nodes it is 392, ...

Reply:
In my view, `hold` is directly influenced by `token_retransmit`, the token timeout which depends on nodes counts would calculate out new retransmit timeout. But once the retransmit timeout specified value, the `hold` won't be influenced by nodes count.

Reply:
Yep, I think we are on same page. `hold` depends on `token_retransmit` which depends on `token_timeout`. But your wording is 

> The hold will be automatically calculated if token_retransmit is modified.

That is just half true. Hold is always automatically calculated and it is most notably affected by number of nodes (because people usually don't change other values, like `token_retransmit`).

Reply:
Got your point. Thanks.